.github/workflows/test.yml

```
@@ -6,7 +6,7 @@ on:
       node-version:
         description: Node.js version
         required: false
-        default: '["14", "16", "18", "20"]'
+        default: '[16", "18", "20"]'
```

```suggestion
        default: '["18", "20", "current"]'
```

Shall we roll the 18+ and "current" changes into one too? It seems like a sensible default that we can override on a per repo-basis.